native-image-workshop/README.md

```diff
@@ -96,11 +104,13 @@ $ ls ./target
 
 The file you will need is, `graalvmnidemos-1.0-SNAPSHOT-jar-with-dependencies.jar`.
 
+**NOTE** : You will need to have installed the GraalVM Native Image package for GraalVM - see the set up instructions.
```

Comment:
This needs some rephrasing:
_NOTE: Make sure the Native Image support is installed for GraalVM - see the set up instructions._
OR maybe:
_NOTE: You need to have Native Image plugin installed in GraalVM - see the set up instructions._

Reply:
updated, used your second suggestion